.github/workflows/standalone-fleet-ci.yml

```diff
@@ -0,0 +1,119 @@
+name: E2E Standalone Fleet
+
+on:
+  schedule:
+    - cron:  '0 5 30 * *'
```

Comment:
why do we need a cron schedule?

Reply:
It will give us a signal on flakiness of the tests and the health of the main branch. Especially in times of low activity, it's good to see that the main branch is still green. It's also a good baseline to judge a PR's status by, if nightly is red and a PR is green, that creates doubt. Finally, it can indicate problems with dependencies in the workflow.